docs/tutorial/03_deploying_magma_access_gateway.md

```diff
@@ -0,0 +1,115 @@
+# 3. Deploying Magma Access Gateway
+
+## Create an EC2 instance on AWS
+
+### Create a security group for allowing SSH access
+
+Retrieve your default VPC ID:
+
+```console
+aws ec2 describe-vpcs --filters "Name=is-default,Values=true" --query 'Vpcs[*][VpcId]' --output text
+```
+
+Create a security group using this VPC ID:
+
+```console
+aws ec2 create-security-group --group-name "demo" --description "Allow SSH access" --vpc-id <your VPC ID>
+```
+
+Note the `GroupId` and use it to add a rule that allows ssh access:
+
+```console
+aws ec2 authorize-security-group-ingress --group-id <security group ID> --protocol tcp --port 22 --cidr x.x.x.x/x
```

Comment:
Do we want to leave the CIDR as a choice `x.x.x.x/x`? Probably it would be better to provide a working CIDR even if it's just allowing all ips